admin/tests/unit/routes/authenticated/certifications/certification/neutralization-test.js

```diff
@@ -0,0 +1,11 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Route | authenticated/certifications/certification/neutralization', function(hooks) {
+  setupTest(hooks);
+
+  test('it exists', function(assert) {
+    const route = this.owner.lookup('route:authenticated/certifications/certification/neutralization');
+    assert.ok(route);
+  });
+});
```

Comment:
Tu peux supprimer ce test. C'est une page en chantier, on ajoutera des tests quand on aura quelque chose à tester. Ce test est, je crois, le test par défaut généré par ember lorsqu'on créé une nouvelle route. Je crois d'ailleurs aussi qu'ils y mettent un commentaire style // replace this test with your tests, donc là vu qu'on n'a rien encore je propose de carrément supprimer ce fichier